components/input/input.html

```diff
@@ -0,0 +1,149 @@
+                    <a class="side-nav" href="../avatar/avatar.html">Avatar</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav" href="../badge/badge.html">Badge</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav" href="../button/button.html">Button</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav" href="../card/card.html">Card</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav" href="../image/image.html">Image</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav link-active" href="../input/input.html">Input</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav" href="../lists/lists.html">Lists</a>
+                </li>
+                <li class="nav-content">
+                    <a class="side-nav" href="../rating/rating.html">Rating</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <!-- input -->
+    <main class="page-content">
+        <section class="section-input">
+            <h1 class="head-topic">Input</h1>
+            <p class="section-about">
+                Inputs are used to take the info by user.
+            </p>
+
+            <h2 class="sub-header">Textbox</h2>
+            <p class="section-about">The textbox is a multi-line text input control. It is widely used in a form, to collect user inputs like comments or feedbacks. 
+            <br>To use textarea, use the class <span>textbox</span>.
+            </p>
+            <div class="textbox-input">
+                <textarea class="textarea"></textarea>
+            </div>  
+            <iframe class="textarea-iframe" src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ctextarea%2520class%253D%2522textarea%2522%253E%253C%252Ftextarea%253E%250A" style="width: 455px; height: 207px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <div class="divider"></div>
+
+            <h2 class="sub-header">Label Input</h2>
+            <p class="section-about">Label Input has label present on top of the input field. This signifies the user of the value to be entered in respective fields.</p>
+            <div class="label-input">
+                <div class="input-box">
+                <label for="username" class="input label">Username</label>
+                <input type="text" class="input">
+                </div>
+                <div class="input-box">
+                <label for="password" class="input label">Password</label>
```

Comment:
This is incorrect. To bind the label to the input tag, the **id** of the input tag is assigned to the for attribute.